versions.mk

```diff
@@ -24,7 +24,11 @@ vVERSION := v$(VERSION:v%=%)
 GOLANG_VERSION ?= 1.20.4
 CUDA_VERSION ?= 11.8.0
 
+CLIENT_GEN_VERSION ?= v0.26.1
+CONTROLLER_GEN_VERSION ?= v0.9.2
```

Comment:
Do we need these again now since we are still based off of the golang image:
```
 GOLANGCI_LINT_VERSION ?= v1.52.0
MOQ_VERSION ?= v0.3.4
```

Reply:
comment added